lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -65,7 +65,7 @@ sub _git_clone ($app, $job, $ctx, $path, $url) {
 
         my $origin_url = $git->get_origin_url;
         if ($url ne $origin_url) {
-            $ctx->warn("Local checkout at $path has origin $origin_url but requesting to clone from $url");
+            $ctx->info("Local checkout at $path has origin $origin_url but requesting to clone from $url");
```

Comment:
Can you rephrase the message to make the "so what" clear?

Reply:
Is this actionable? My understanding from the ticket is this works as expected. So I would actually say remove it.

Reply:
This can be helpful without being actionable, e.g. when debugging and trying to make sense of what our Git code does or skips.